lib/OpenQA.pm

```diff
@@ -39,7 +39,7 @@ sub _read_config {
             hsts => 365,
         },
         auth => {
-            method => 'OpenID',
+            method => 'Fake',
```

Comment:
Just to document offline discussion: We need to change this at least in the package. The code (not just a config file template) must default to openid there.